specification/purview/data-plane/Azure.Analytics.Purview.Scanning/preview/2022-07-01-preview/scanningService.json

```diff
@@ -0,0 +1,9155 @@
+        "x-ms-examples": {
+          "Filters_CreateOrUpdate": {
+            "$ref": "./examples/Filters_CreateOrUpdate.json"
+          }
+        }
+      }
+    },
+    "/integrationruntimes/{integrationRuntimeName}": {
+      "get": {
+        "tags": [
+          "IntegrationRuntimes"
+        ],
+        "description": "Gets an integrationRuntime",
+        "operationId": "IntegrationRuntimes_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "integrationRuntimeName",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/IntegrationRuntime"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Scanning Service",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "IntegrationRuntimes_Get": {
+            "$ref": "./examples/IntegrationRuntimes_Get.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "IntegrationRuntimes"
+        ],
+        "description": "Deletes an integrationRuntime",
+        "operationId": "IntegrationRuntimes_Delete",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "integrationRuntimeName",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/IActionResult"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Scanning Service",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "IntegrationRuntimes_Delete": {
+            "$ref": "./examples/IntegrationRuntimes_Delete.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "IntegrationRuntimes"
+        ],
+        "description": "Creates or Updates an instance of a integrationRuntime",
+        "operationId": "IntegrationRuntimes_CreateOrUpdate",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "integrationRuntimeName",
+            "required": true,
+            "type": "string"
```

Comment:
Path parameter should specify a maximum length (maxLength) and characters allowed (pattern).

Reply:
Added